code/Test_definitions/Traffic_Influence.yaml

```diff
@@ -0,0 +1,1030 @@
+---
+openapi: 3.0.3
+############################################################################
+#                                     API info                             #
+############################################################################
+info:
+  title: Traffic Influence API
+  version: wip
+  description: |
+    ## Overview
+    The reference scenario foresees a Service, composed by one or more Service
+    Producers deployed in different geographical locations on Edge Cloud Zones
+    (Edge datacentres of Telco Operator) or in Cloud. The Service Producer,
+    deployed at the Edge, is referred as Edge Application Server (EAS).\
+    An Edge Cloud Zone is a platform in the Telco Operator network, offering
+    network, compute and storage resources to developers. A developer can
+    deploy and run applications on an Edge Cloud Zone, meaning reduced latency
+    to end users that are nearby, as the network path is shorter. A network
+    operator's EdgeCloud may comprise multiple Edge Cloud Zones, each in a
+    discrete location to bring latency benefits to end users across a country.\
+    The operator can help developers know which of the Edge Cloud Zones will
+    bring the best performance benefit for a given end user and application\.
+    The Traffic Influence API (TI API) provides the fastest routing from the
+    user Device (e.g. a Smartphone) to the optimal EAS instance in a specific
+    geographical location, installed in an Edge Cloud Zone.\
+    If a Service is offered by Cloud Instances and by Edge Instances, the TI API
+    can be used get the optimal routing of the traffic to the Edge Instances,
+    maybe for a set of users. Getting the optimal routing can be used to improve
+    latency maybe in combination with other CAMARA APIs such as QoD (Quality On
+    Demand). Providing the optimal routing is indeed an important step to get
+    the optimal latency.\
+    If the TI API is used to get the best routing at the Edge for a Device in a
+    geographical location and the Device moves to another geographical location,
+    the TI API can be invoked to get the optimal routing in the new geographical
+    location for that Device.
+    ## Introduction
+    The TI API provides the capability to establish the best routing, in terms
```

Comment:
I think we have changed the best to optimal for other APIs so may be here also we can use optimal in place of best.